src/__tests__/wrap.e2e.test.ts

```diff
@@ -1,4 +1,5 @@
 import { wrap } from "../implementations/wrap";
```

Comment:
I think there are missing cases

![image](https://user-images.githubusercontent.com/503185/79829229-13b31e80-83d5-11ea-8e61-e30c62cdd7a0.png)

Are you sure the 5 cases are covered ?

Reply:
yes i have tested all

Reply:
i can only test the function of wrap,  but i cannot test cli input which covers the case of input is directory with other output types.

Reply:
i will add test for output type of std out